testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/multinode-all/snowflake.properties

```diff
@@ -0,0 +1,7 @@
+connector.name=snowflake
+snowflake.test.server.url=jdbc:snowflake://<account>.snowflakecomputing.com
+snowflake.test.server.user=root
+snowflake.test.server.password=secret
+snowflake.test.server.database=database
+snowflake.test.server.role=role
+snowflake.test.server.warehouse=warehouse
```

Comment:
Property names are wrong. Also, please set values via environment variables. Another file is also. 
```
${ENV:SNOWFLAKE_JDBC_URL}
${ENV:SNOWFLAKE_USER}
${ENV:SNOWFLAKE_PASSWORD}
${ENV:SNOWFLAKE_DATABASE}
${ENV:SNOWFLAKE_ROLE}
${ENV:SNOWFLAKE_WAREHOUSE}
```